src/components/organisms/Accordion/AccordionHeader/AccordionHeader.md

```diff
@@ -12,13 +12,13 @@
 ```js
 import { Accordion } from '@zopauk/react-components';
 
-<Accordion.Header isOpen={true}>opened accordion header</Accordion.Header>;
+// <Accordion.Header isOpen={true}>opened accordion header</Accordion.Header>;
```

Comment:
is this intended?

Reply:
nope, was work in progress before, fixed now